full-stack-tests/imodelhub-client/src/imodelhub/UrlValidator.test.ts

```diff
@@ -106,7 +106,7 @@ describe.skip("Validate iModelHub URL Whitelist", () => {
     // filter out duplicate URLs by putting the lines in a set and create an array from it again
     const loggedUrls: string[] = Array.from(new Set(logFileContent.split(/\r?\n/)));
 
-    let baseUrl: string = await new UrlDiscoveryClient().discoverUrl(new ClientRequestContext(), IModelBaseHandler.searchKey, undefined);
+    let baseUrl: string = (IModelBaseHandler as any).baseUrl; //  await new UrlDiscoveryClient().discoverUrl(new ClientRequestContext(), IModelBaseHandler.searchKey, undefined);
```

Comment:
I think using baseUrl is fine here. We should probably removed that test altogether, since it was meant to track API changes when iModelBank was initially developed.

Reply:
Okay, I'll make a note and I can fix this in a follow up PR.